src/01-functions.js

```diff
@@ -74,7 +82,9 @@ function multiply() {}
  *  average(10, 6);
  *  //> 8 // 10 + 6 = 16; 16/2 = 8;
  */
-function average() {}
+function average(a, b) {
+  return ((a+b) / 2);
```

Comment:
I would space out the arithmetic assignment to make it slightly easier to read.  Also we don't need the surrounding `()` in this case.